README.md

```diff
@@ -7,7 +7,7 @@
 <a href="https://packagist.org/packages/laravel/framework"><img src="https://img.shields.io/packagist/l/laravel/framework" alt="License"></a>
 </p>
 
-## About Laravel
+## About Laravel 2
```

Comment:
Mistake?

Reply:
Intentional to see the effect of the cache.